_data/meltano/utilities/tableau/tableau.yml

```diff
@@ -0,0 +1,52 @@
+commands:
+  refresh:
+    args: refresh $TABLEAU_DATASOURCE_LUID
```

Comment:
@janhoon is this input something that you could optionally include as a setting? It might be nice for a user to configure their project with the ID in the meltano.yml vs retrieving it from the environment.

Reply:
included it is a setting, pr for support for this is open on the extension as well